### ISSUE TITLE
[Community members list] fix right click behavior

### ISSUE THREAD

**Issue by alexandraB99** [status-im/status-desktop#7298]
### Description

RIght click in members list in community doesn’t work (should show profile context menu)

**Comment by faust4exe** [comment]
Issue fixed by another PR https://github.com/status-im/status-desktop/pull/7329
This are menus on master after it is merged
```
commit 326e9e09a56b2e27b61b856434c8fc383115266f
Date:   Mon Sep 12 18:01:25 2022 +0200
    fix: unbreak User List Panel context menu
    StatusAssetSettings porting leftover
    Closes #7302
```
![Screenshot from 2022-09-12 21-18-10](https://user-images.githubusercontent.com/6445843/189728203-d1efb9c2-26cf-4ee7-b355-7ad5122a443d.png)
![Screenshot from 2022-09-12 21-18-41](https://user-images.githubusercontent.com/6445843/189728240-cc44683c-7990-459d-acfb-55c128a4667b.png)

**Comment by caybro** [comment]
Yeah, fixed
